mne/channels/channels.py

```diff
@@ -644,10 +644,9 @@ def pick_types(self, meg=True, eeg=False, stim=False, eog=False,
         Parameters
         ----------
         meg : bool | str
-            If True include all MEG channels. If False include None.
-            If string it can be 'mag', 'grad', 'planar1' or 'planar2' to select
-            only magnetometers, all gradiometers, or a specific type of
-            gradiometer.
+            If True include MEG channels. If string it can be 'mag', 'grad',
+            'planar1' or 'planar2' to select only magnetometers, all
+            gradiometers, or a specific type of gradiometer.
```

Comment:
```suggestion
            If ``True``, include MEG channels. If string, it can be ``'mag'``, 
            ``'grad``', ``'planar1'`` or ``'planar2'`` to select only magnetometers
             all gradiometers, or a specific type of gradiometer.
```